.github/workflows/ci-test.yml

```diff
@@ -27,5 +27,4 @@ jobs:
         hugo version
 
     - name: Build
-      run: |
-        make build
+      run: |
```

Comment:
why do you need to change this file?

Reply:
I misunderstood what was asked. I reverted the change to the file by reverting back to the commit `Disable links checking...` on the ci-test.yml file.